src/lib/date-helper.js

```diff
@@ -19,11 +51,12 @@ const formatDateFromApiInline = date => {
 }
 
 const formatDateToApi = date => {
-  const timestamp = date && date.valueOf()
+  const timestamp = date && dateToTimeStamp(new Date(date)).valueOf()
```

Comment:
why do you need dateToTimeStamp then new Date here?

Reply:
this function is added for time shifting keeping into consideration formats